src/main/java/edu/agh/wfiis/designpatterns/patternsUsageExample/design_patterns/BuilderDesignPattern.java

```diff
@@ -0,0 +1,28 @@
+package edu.agh.wfiis.designpatterns.patternsUsageExample.design_patterns;
+
+public class BuilderDesignPattern extends DesignPattern {
+    @Override
+    public String getName() {
+        return "Builder";
+    }
+
+    @Override
+    public Type getType() {
+        return Type.CREATIONAL;
+    }
+
+    @Override
+    public String getDescription() {
+        return "some builder desc";
+    }
+
+    @Override
+    public String getExample() {
+        return "some builder example";
+    }
+
```

Comment:
This approach makes it impossible to load informations like description and example from external source. It must be hardcoded in class body = Open Closed principle violation.
Each little change in description string = modifying class source.
Moreover a real getExample string would be multiple line and much longer, which would look really bad in class source.

Reply:
Having 30 design pattern in the system would also entail having 30 files with their classes, so the project's structure would be a bit of a mess